src/index.html

```diff
@@ -4,9 +4,352 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="./images/logo.png" alt="moyo_logo">
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#phones" class="nav__link">
+              смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops" class="nav__link" data-qa="nav-hover">
```

Comment:
3 and more properties should be as column, not row
```suggestion
            <a 
              href="#laptops" 
              class="nav__link" 
              data-qa="nav-hover"
            >
```